spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/influx/InfluxDbProperties.java

```diff
@@ -67,4 +102,156 @@ public void setPassword(String password) {
 		this.password = password;
 	}
 
+	public InfluxDB.ConsistencyLevel getConsistency() {
+		return this.consistency;
+	}
+
+	public void setConsistency(InfluxDB.ConsistencyLevel consistency) {
+		this.consistency = consistency;
+	}
+
+	public String getDatabase() {
+		return this.database;
+	}
+
+	public void setDatabase(String database) {
+		this.database = database;
+	}
+
+	public InfluxDB.LogLevel getLog() {
+		return this.log;
+	}
+
+	public void setLog(InfluxDB.LogLevel log) {
+		this.log = log;
+	}
+
+	public String getRetentionPolicy() {
+		return this.retentionPolicy;
+	}
+
+	public void setRetentionPolicy(String retentionPolicy) {
+		this.retentionPolicy = retentionPolicy;
+	}
+
+	public boolean isGzipEnabled() {
+		return this.gzipEnabled;
+	}
+
+	public void setGzipEnabled(boolean gzipEnabled) {
+		this.gzipEnabled = gzipEnabled;
+	}
+
+	public Batch getBatch() {
+		return this.batch;
+	}
+
+	public static class Batch {
+
+		/**
+		 * Whether to enable Batch configuration.
+		 */
+		private boolean enabled;
+
+		/**
+		 * Number of actions to collect.
+		 */
+		private int actions = 1000;
+
+		/**
+		 * Time to wait in milliseconds.
+		 */
+		private int flushDuration = 1000;
```

Comment:
`flushDuration` and `jitterDuration` are set as integers with `TimeUnit.MILLISECONDS` so that the reason both are provided with such descriptions

See https://github.com/influxdata/influxdb-java/blob/master/src/main/java/org/influxdb/impl/InfluxDBImpl.java#L308

Reply:
It doesn't matter. Most `Duration` we have in `ConfigurationProperties` are actually mapped to int/long values (check existing keys).

Reply:
I had forgotten `@DurationUnit` 😅

Reply:
DurationUnit has nothing to do with what we're discussing here IMO. I don't understand why you bring this up.